discordBot/main.py

```diff
@@ -0,0 +1,223 @@
+from discord.ext import commands
+from discord import Embed, File, TextChannel, Status, CustomActivity, ActivityType, Invite
+from datetime import timedelta
```

Comment:
Already imported below:
https://github.com/Coding-Web-Community/CodingBump/blob/8576c399671653578cee92ea1aaa6b65d4b1e649/discordBot/main.py#L9

Reply:
Ah yes, I'll remove that